src/components/pages/block-txs/block-txs-table-row.tsx

```diff
@@ -32,20 +33,16 @@ const BlockTxsTableRow = ({
       </TableCell>
       <TableCell>{timestampFormattedAsDate ? utc : distance}</TableCell>
       <TableCell className="max-w-40 truncate text-primary hover:brightness-150">
-        <Link
+        <AddressLink
```

Comment:
To resolve the conflicts with main, you will need to swap the `Link` with the `AddressLink` component and make sure `AddressLink` takes the exact same className prop as `Link`.